helpers/localStorage.js

```diff
@@ -0,0 +1,45 @@
+import { useCallback, useEffect, useState } from "react";
+
```

Comment:
Why did you choose to use this local storage hook and did not work with `local-storage-state`? ([See handout for the local storage session) ](https://github.com/neuefische/web-muc-23-1/blob/main/sessions/react-with-local-storage/react-with-local-storage.md#react-with-local-storage-1)🤔